### PR TITLE
Upgrade to Django 2.0.5

### LINKED ISSUE #184: Upgrade to Django 2.0
https://docs.djangoproject.com/en/2.0/releases/2.0/

### PR DESCRIPTION
**Pull request by praseodym**
Closes #184

### CODE REVIEW

File: ldb/models.py [hunk 1]
```diff
@@ -96,15 +96,15 @@ class Organization(Entity):
...
 
     name_prefix = models.CharField(_('name prefix'), max_length=100, blank=True)
     name = models.CharField(_('name'), max_length=100)
     name_short = models.CharField(_('name short'), max_length=100, blank=True)
     salutation = models.CharField(_('salutation'), max_length=100)
 
-    @models.permalink
     def get_absolute_url(self):
-        return 'ldb_organizations_detail', [str(self.id)]
+        return reverse('ldb_organizations_detail', args=[str(self.id)])
```
Comment (jgadelange) [inline ldb/models.py]: Hmmm... why the `str()`?

**Comment by jgadelange** [comment]
Oh, and apparently django 2.0.5 is out ;P

File: ldb/models.py (hunk 1 above)
Comment (praseodym) [reply ldb/models.py]: Because it was already there, I presume. Do we want to change it in this PR?

Comment (jgadelange) [reply ldb/models.py]: Nope

**Pull request merged**